tests/integration/conftest.py

```diff
@@ -17,6 +17,11 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+# Test tiers:
+#   * Tier 1: Normal test case and should never fail.
+#   * Tier 2: Decorative test case and should never fail.
```

Comment:
What do you meant with decorative?

Reply:
Oh. I forgot the remove them.

Current PR is only choosing the test cases for tier1, don't want to discuss on other tiers.